File: 백준/Bronze/4589. Gnome Sequencing/Gnome Sequencing.c
#include <stdio.h>

int main()
{
    int n, a, b, c;

    scanf("%d", &n);
    printf("Gnomes:\n");

    for (int i = 0; i < n; i++) {
        scanf("%d %d %d", &a, &b, &c);
        if ((a <= b && b <= c) || (c <= b && b <= a))
            printf("Ordered\n");
        else
            printf("Unordered\n");
    }

    return 0;
}

File: 백준/Bronze/4589. Gnome Sequencing/Gnome Sequencing.py
n = int(input())
print("Gnomes:")
for i in range (n):
    a, b, c = map(int, input().split())
    if (a <= b and b <= c) or (c <= b and b <= a):
        print("Ordered")
    else:
        print("Unordered")